src/components/StudentList/StudentList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { StudentCard } from "..";
+import { Student } from "../../interfaces";
+import { VirtualizedList } from "../reusables";
+
+interface StudentListProps {
+  filteredStudents: Student[];
+  handleStudentDialogOpen: () => void;
+}
+
+export const StudentList: React.FC<StudentListProps> = ({ filteredStudents, handleStudentDialogOpen }) => {
+  return (
+    <VirtualizedList idPath="epId" listData={filteredStudents} overscan={500}>
```

Comment:
For this virtualization library, is `overscan` in pixels or in number of elements in a list?

If it's number of elements, you might be able to switch it to something like `10` and have better performance!

Reply:
It’s in pixels. I was very sad when I realized it was pixels instead of number of elements 😢